### PR TITLE
Range slider

### PR DESCRIPTION
**Pull request by aVileBroker**
Feedback welcome on structure, types, everything :D (even the name)

### CODE REVIEW

**Comment by joien1** [comment]
Did you mean to include the package.lock file?

File: packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+    transition: filter .1s;
+
+    ${
+      disabled
+        ? `
+      filter: grayscale(1) contrast(.5) brightness(1.2);
+      pointer-events: none;
+    `
+        : ''
+    }
```
Comment (joien1) [inline packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx:45]: Not sure if prettier went ham, but there's a lot of unneeded line breaks here.

File: packages/hs-react-ui/src/components/RangeSlider/types.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  motionBlur?: boolean;
+  springOnRelease?: boolean;
+  debounceInterval?: number;
+  axisLock?: 'x' | 'y' | '';
+  onDrag?: (val) => void;
+  disabled?: boolean;
+  min: number;
+  max: number;
+  values?: number[] | valueProp[];
+  testId: string;
```
Comment (joien1) [inline packages/hs-react-ui/src/components/RangeSlider/types.ts:45]: Imo should be optional, maybe with a sensible default

File: packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx (hunk 1 above)
Comment (aVileBroker) [reply packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx:45]: It really went crazy

File: packages/hs-react-ui/src/components/RangeSlider/types.ts (hunk 2 above)
Comment (aVileBroker) [reply packages/hs-react-ui/src/components/RangeSlider/types.ts:45]: oops yes

Comment (aVileBroker) [reply packages/hs-react-ui/src/components/RangeSlider/types.ts:45]: I don't think it needs a default as this just gets appended to the default... maybe it should be called testIdSuffix or something lol

**Pull request merged**